src/ITensorNetworks.jl

```diff
@@ -49,6 +50,9 @@ include("solvers/solver_utils.jl")
 include("solvers/defaults.jl")
 include("solvers/insert/insert.jl")
 include("solvers/extract/extract.jl")
+include("solvers/subspace_expansions/linalg/standard_svd.jl")
+#include("solvers/subspace_expansions/linalg/rsvd_aux.jl")
+#include("solvers/subspace_expansions/linalg/rsvd_linalg.jl")
```

Comment:
Why is this commented out?

Reply:
To be deleted, as discussed in https://github.com/mtfishman/ITensorNetworks.jl/pull/137/#issuecomment-2080151198.